Dockerfile

```diff
@@ -1,30 +1,36 @@
 FROM python:3.5.4-alpine
-ENV PYTHONUNBUFFERED 1
-
-# set timezone
-RUN apk update && apk add --no-cache tzdata && \
-    cp /usr/share/zoneinfo/America/Sao_Paulo /etc/localtime && \
-    echo "America/Sao_Paulo" >  /etc/timezone
-
-# pyscopg2 dependencies
-RUN apk update && apk add --no-cache \
-    gcc \
-    musl-dev \
-    postgresql-dev \
-    python3-dev
-
-# install git so we can pip install from github repos
-RUN apk update && apk add --no-cache git
-
-WORKDIR /code
+ENV PYTHONUNBUFFERED=1 \
+    GOSS_VERSION=${GOSS_VERSION:-v0.3.5}
```

Comment:
❤️  Goss

Reply:
Forgot to mention the size reduction. From 373M to 178M